app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -7,6 +7,9 @@
 
 import java.util.List;
 
+/**
+ * /tasks 에 대한 사용자 요청을 처리합니다.
+ */
```

Comment:
`/tasks`에 커플링이 생겨 아쉽습니다. 이렇게 표현하면 어떨까요? 
```suggestion
/**
 * 할 일에 대한 HTTP 요청의 처리를 담당합니다.
 */
```